angular/src/lib/date-picker/date-picker-day.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit, Input, HostBinding, HostListener, ElementRef } from '@angular/core';
+import { DatePickerService } from './date-picker.service';
+
+@Component({
+  selector: 'cui-date-picker-day',
+  template: `
+    {{ displayDay }}
+  `,
+  styles: []
+})
+export class DatePickerDayComponent implements OnInit {
+
+  /** @prop Required day that the DatePickerDay displays */
+  @Input() day: any;
+  /** @prop Required month that the DatePickerDay displays */
+  @Input() month: any;
+
+  private isDisable = false;
+  private isSelected = false;
+  private isFocus = false;
+  private isToday = false;
+  private isOutsideMonth = false;
+
+  public displayDay;
+
+  @HostBinding('class') get classes(): string {
+    return 'cui-button'+
+    ' cui-button--circle cui-button--28' +
+    `${(this.isDisable && ` cui-button--disabled`) || ''}` +
+    ' cui-datepicker__day'+
+    `${(this.isSelected && ` cui-datepicker__day--selected`) || ''}` +
+    `${(this.isFocus && ` cui-datepicker__day--focus`) || ''}` +
+    `${(this.isToday && ` cui-datepicker__day--today`) || ''}` +
+    `${(this.isOutsideMonth && ` cui-datepicker__day--outside-month`) || ''}` +
+    ``;
+  };
```

Comment:
So i believe we are going to use the updated class pattern here and apply the static css classes as well as boolean css to the host inside @Component({}). @pauljeter